monkey/infection_monkey/post_breach/actions/communicate_as_backdoor_user.py

```diff
@@ -49,11 +49,18 @@ def run(self):
                     )
                 )
                 exit_status = new_user.run_as(http_request_commandline)
-                self.send_result_telemetry(exit_status, http_request_commandline, username)
+                result = self._get_result_for_telemetry(
+                    exit_status, http_request_commandline, username
+                )
+                # `command` is empty here; we could get the command from `new_user` but that
+                # doesn't work either since Windows doesn't use a command, it uses win32 modules
+                self.pba_data.append(PostBreachData(self.name, "", result))
```

Comment:
Same as previous comment, I'd suggest we use `self.command`, this way if we need to change the default empty value we can do it in one place